### ISSUE TITLE
Can't install 1.4.x on PG 9.6

### ISSUE THREAD

**Issue by jamatthews** [postgrespro/pg_pathman#101]
I have pg_pathman 1.3.1 installed on a PG 9.6.1 DB and would like to upgrade to 1.4.1

```
* Restarting PostgreSQL 9.6 database server                                                                                                                                                                                                                                     * The PostgreSQL server failed to start. Please check the log output:
2017-07-10 10:12:30 UTC FATAL:  could not load library "/usr/lib/postgresql/9.6/lib/pg_pathman.so": /usr/lib/postgresql/9.6/lib/pg_pathman.so: undefined symbol: convert_tuples_by_name_map
```

It seems like convert_tuples_by_name_map was introduced in PG10 but there's no version check around this code? https://github.com/postgrespro/pg_pathman/blob/a4e190a5a8560c04d5a8ba10c4e14dd884c26be7/src/pl_funcs.c#L663

Thanks

**Comment by lmyyao** [comment]
i install pg_patthman on PG9.5.5 but always report 
could not load library "/usr/lib/postgresql/9.5/lib/pg_pathman.so": /usr/lib/postgresql/9.5/lib/pg_pathman.so: undefined symbol: convert_tuples_by_name_map

**Comment by jamatthews** [comment]
Does PG actually start?

**Comment by jamatthews** [comment]
I just discovered it actually works on 9.6.3 but definitely not 9.6.1.

**Comment by funbringer** [comment]
Hi @jamatthews, @lmyyao 

Actually, the problem is that some functions (e.g. `convert_tuples_by_name_map`) were deleted from header in 9.6.1, but have been restored in 9.6.3. You won't be able to use pg_pathman with 9.6.1.

**Comment by funbringer** [comment]
> It seems like convert_tuples_by_name_map was introduced in PG10 but there's no version check around this code?

That's not true, branches `REL9_5_STABLE`, `REL9_6_STABLE`, `master` contain these functions.

**Comment by zilder** [comment]
Hi @jamatthews,

We'll provide a compatibility fix for older versions of Postgres shortly

**Comment by zilder** [comment]
@jamatthews, @lmyyao, could you please switch to `hotfix-101` branch and then try to build and install pg_pathman from there? Does it solve the issue?

**Comment by jamatthews** [comment]
@funbringer @zilder Thank you for the incredibly quick clarification!

**Comment by lmyyao** [comment]
@funbringer @zilder  it works. Thank you very much!

**Comment by funbringer** [comment]
Merged into `master`, we'll roll out 1.4.2 soon enough.
